04-onion-routing.md

```diff
@@ -280,7 +280,9 @@ The transformations at hop `k` are as follows:
 
  - The shared secret `ss_k` is computed by first blinding the hop's public key
  `nodepk_k` with all previous blinding factors `{b_1, ..., b_{k-1}}` (if any),
- and second, executing ECDH with the blinded public key and the `sessionkey`.
+ and second, executing ECDH with the blinded public key and the `sessionkey` to
+ obtain a group element. The shared secret `ss_k` is then calculated as the
+ `SHA256` hash of the ECDH output, serialized in the compressed format.
```

Comment:
This is explained further down in the [Shared Secret](https://github.com/jimpo/lightning-rfc/blob/ac143c18533752124b09956a02cfb4563cb08051/04-onion-routing.md#shared-secret) section. I'd prefer having a reference to that part rather than duplicating the instructions.